service/storage_service.cc

```diff
@@ -6179,6 +6179,24 @@ void storage_service::init_messaging_service(bool raft_topology_change_enabled)
                             rs->partitions(), std::back_inserter(cdc_generation_mutations), [s] (const partition& p) {
                         return canonical_mutation{p.mut().unfreeze(s)};
                     });
+
+                    // Copy mutations n times, where n is picked so that the memory size of all mutations
```

Comment:
I'd put the comment inside the injection so it's more clear when reading top-to-bottom that this is error injection specific logic.